appveyor.yml

```diff
@@ -17,7 +17,9 @@ install:
   - npm install -g npm@5.5.1 yarn@1.3.2
   - pip install virtualenv==15.1.0
   - cinst sbt --version 1.0.2 -y
+  - C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Sy cvs"
```

Comment:
Just FYI: I just double-checked that there's indeed [no Chocolatey package for CVS](https://chocolatey.org/packages?q=cvs) that we could use.